hack/ci/e2e-k8s.sh

```diff
@@ -73,33 +82,24 @@ create_cluster() {
   CLUSTER_LOG_FORMAT=${CLUSTER_LOG_FORMAT:-} 
   # potentially enable --logging-format
   kubelet_extra_args="      \"v\": \"${KIND_CLUSTER_LOG_LEVEL}\""
-  if [ -n "${KUBELET_LOG_FORMAT:-$CLUSTER_LOG_FORMAT}" ]; then
-    case "${KUBE_VERSION}" in
-     v1.1[0-8].*)
-      echo "KUBELET_LOG_FORMAT is only supported on versions >= v1.19, got ${KUBE_VERSION}"
-      exit 1
-      ;;
-    *)
-      # NOTE: the indendation on the next line is meaningful!
+  KUBELET_LOG_FORMAT=${KUBELET_LOG_FORMAT:-$CLUSTER_LOG_FORMAT}
+  if [ -n "$KUBELET_LOG_FORMAT" ]; then
+      check_log_format
       kubelet_extra_args="${kubelet_extra_args}
       \"logging-format\": \"${KUBELET_LOG_FORMAT}\""
-      ;;
-    esac
   fi
 
   scheduler_extra_args="      \"v\": \"${KIND_CLUSTER_LOG_LEVEL}\""
-  if [ -n "${SCHEDULER_LOG_FORMAT:-$CLUSTER_LOG_FORMAT}" ]; then
-    case "${KUBE_VERSION}" in
-     v1.1[0-8].*)
-      echo "SCHEDULER_LOG_FORMAT is only supported on versions >= v1.19, got ${KUBE_VERSION}"
-      exit 1
-      ;;
-    *)
-      # NOTE: the indendation on the next line is meaningful!
+  controllerManager_extra_args="      \"v\": \"${KIND_CLUSTER_LOG_LEVEL}\""
+  apiServer_extra_args="      \"v\": \"${KIND_CLUSTER_LOG_LEVEL}\""
+  if [ -n "$CLUSTER_LOG_FORMAT" ]; then
+      check_log_format
```

Comment:
this shouldn't be throwing an error message about KUBELET_LOG_FORMAT

Reply:
Fixed in the latest commit.